pypdf/filters.py

```diff
@@ -501,6 +502,19 @@ def group(self) -> int:
         return CCITTgroup
 
 
+def __create_old_class_instance(
+    K: int = 0,
+    columns: int = 0,
+    rows: int = 0
+) -> CCITTParameters:
+    deprecate_with_replacement("CCITParameters", "CCITTParameters", "5.0.0")
```

Comment:
The message looks wrong:

> CCITParameters is deprecated and will be removed in pypdf 5.0.0. Use CCITTParameters instead.

The current release is 5.1.0.

Reply:
Is obvious (now!). Will change to 5.2.0. Plus add a test of the original.

Reply:
Shouldn't it still be 6.0.0 instead? We will surely not remove this in the next patch release.